regression/contracts/assigns-enforce-malloc-zero/main.c

```diff
@@ -6,7 +6,7 @@ int foo(char *a, int size)
   // clang-format off
   __CPROVER_requires(0 <= size && size <= __CPROVER_max_malloc_size)
   __CPROVER_requires(a == NULL || __CPROVER_is_fresh(a, size))
-  __CPROVER_assigns(a: __CPROVER_POINTER_OBJECT(a))
+  __CPROVER_assigns(a: __CPROVER_whole_object(a))
```

Comment:
I would really like this to be named "__CPROVER_object_whole" for consistency with "__CPROVER_object_from" and "__CPROVER_object_slice"